### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**

### CODE REVIEW

**Comment by nathan-contino-mongo** [comment]
@cmelchior requesting review

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Realm is a mobile database that runs directly inside phones, tablets or wearables.
 
-This repository holds adapters for combining Realm Java with Android UI components and framework classes.
+This repository holds adapters for combining the Realm Android SDK with Android UI components and framework classes.
```
Comment (bmunkholm) [inline README.md]: I would suggest linking to realm-java.
But we haven't renamed realm-java to "Realm Android"?

Comment (nathan-contino-mongo) [reply README.md]: Good point on linking to the SDK library -- I think a lot of users who stumble upon this page will appreciate the link.

As for the naming... in the docs, we use the term "Android SDK" to more easily distinguish between realm-swift and realm-java -- the thought is, it's easier for users to find the SDK they care about if they can focus on the platform. It also helps clarify that realm-java only works with Android apps, not arbitrary java applications.

However... those names aren't set in stone. Especially with the Kotlin Multiplatform (or Kotlin Native?) SDK coming up, it might be better to rethink that naming convention, since Android apps will presumably just start using the Kotlin SDK at some point.

I think I'd be more comfortable wording this as "realm-java (also known as the Realm Android SDK)" given our naming convention in the docs. But we're probably long overdue some active thought on the naming convention in the docs... if you feel strongly that we should consider renaming "Android SDK" realm-java, I can make that happen pretty quickly and we can just rephrase this to "Realm Java SDK" or "realm-java" or "Realm Java".

@bmunkholm let me know what you think, great question.

File: README.md [hunk 2]
```diff
@@ -64,24 +66,25 @@ In case you don't want to use the pre-compiled version, you can build the librar
...
 
- * an aar file for the adapter library in `adapters/build/outputs/aar/android-adapters-release.aar`
+ * an `aar` file for the adapter library (`adapters/build/outputs/aar/android-adapters-release.aar`)
 
 ### Other Commands
 
  * `./gradlew monkeyDebug` will run the monkey tests on the example project.
  * `./gradlew javadoc` will create the javadoc for the library.
- * `./gradlew publishToSonatype` will upload artifacts to Maven Central.
+ * `./gradlew artifactoryPublish` will upload a SNAPSHOT to OJO.
```
Comment (cmelchior) [inline README.md]: These are being deleted. When we move to Maven Central, the `publishToSonatype` was correct.

Comment (nathan-contino-mongo) [reply README.md]:
```suggestion
 * `./gradlew publishToSonatype` will upload artifacts to Maven Central.
```
Thanks, fixing.

File: README.md (hunk 1 above)
Comment (nathan-contino-mongo) [reply README.md]:
```suggestion
This repository holds adapters for combining [realm-java](https://github.com/realm/realm-java) (the Realm Android SDK) with Android UI components and framework classes.
```

**Comment by nathan-contino-mongo** [comment]
@cmelchior Responded to the feedback that you and @bmunkholm  provided -- please take a look (and merge this in, if all is well).

File: README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 Realm is a mobile database that runs directly inside phones, tablets or wearables.
 
-This repository holds adapters for combining Realm Java with Android UI components and framework classes.
+This repository holds adapters for combining [realm-java](https://github.com/realm/realm-java) (the Realm Android SDK) with Android UI components and framework classes.
```
Comment (bmunkholm) [inline README.md]:
```suggestion
This repository holds adapters for combining [realm-java](https://github.com/realm/realm-java) (the Realm Java SDK for Android) with Android UI components and framework classes.
```

**Comment by nathan-contino-mongo** [comment]
@cmelchior Can you merge this in if we don't need any more changes?

**Comment by nathan-contino-mongo** [comment]
@cmelchior please merge this in.
